src/main/java/schemacrawler/crawl/StreamlinerSchemaCrawler.java

```diff
@@ -1,12 +1,17 @@
 package schemacrawler.crawl;
```

Comment:
FYI - the only reason this is not in Should be in io.phdata.streamliner is because it needs to access some package-private classes.